index.js

```diff
@@ -102,7 +102,7 @@ module.exports = (config) => {
 
   event.dispatcher.on(event.suite.before, (suite) => {
     recorder.add(async () => {
-      suiteObj = startTestItem(suite.title, rp_SUITE);
+      suiteObj = startTestItem(suite.title, rp_TEST);
```

Comment:
After that changes the structure `TEST -> STEP -> NESTED STEP (if applicable)` will be presented.
Is it a good idea to avoid suites in the reporting structure? I suggest to use the following structure: `SUITE -> TEST (if applicable) -> STEP -> NESTED STEP (if applicable)`, as from reportportal side we expect the top level suite as a container of tests.

Reply:
@m3t4n & @ilangv guys, could you please look at my notes, will it suit you?

Reply:
And what do you think about @DavertMik work on this for Codecept 3.0+
https://github.com/reportportal/agent-js-codecept/pull/3
May be better to implement it on the integration side to avoid additional work with `bootstrapAll` and `teardownAll`?

Reply:
hi @AmsterGet thank you for your attention.
I agree with the suite > test > step > nestedStep structure. However, on the RP side it's represented as LaunchName > Suite > Test > Step > nestedStep and hence I believe @ilangv 's change (i might be wrong). Whereas I agree with you but maybe an improvement point on this is that after Suite the tests can be expanded instead of navigating to a new page (like nested steps do). Personally, I find it hard to reach the steps. 

On the other hand @DavertMik's change seems to be the stable choice. Currently, it is not possible to properly display the parallel executions without any update on this plugin side. The suites are randomly placed under launches and this makes it really hard to see which ones are under which launch. Another option might be appending the suite name to launch id (thinking out loud). 

@DavertMik: any chance on finalizing the PR? CodeceptJS is already 3.2.3.